src/style.css

```diff
@@ -1,3 +1,67 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Consider adding a fallback font family in case the specified font 'Roboto' is not available. For example, 'font-family: Roboto, Arial, sans-serif;' can be written as 'font-family: Roboto, 'Helvetica Neue', Arial, sans-serif;' to provide better cross-platform compatibility.